### DISCUSSION TITLE
Is it possible to use Wavelog and Cloudlog at the same time?

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#371] [General]
Is it possible to use Wavelog and Cloudlog at the same time? For example, on two different domain names, but with database synchronization (station locations, logs and QSOs)?

**Comment by HB9HIL** [comment]
Why would you do that?

And maybe to answer your question. Both Projects had some changes in database structure, so this won't be possible without big issues.

**Reply by HB9HIL** [reply]
to migrate Cloudlog to Wavelog please take a look in this wiki article:

https://github.com/wavelog/Wavelog/wiki/How-to-migrate-Cloudlog-to-Wavelog

**Reply by R1BLH** [reply]
I read about the migration. I'm interested in the possibility to use Wavelog and Cloudlog at the same time with the same database for comparison. Or are they incompatible?
Or may be is possibility for synchronization of databases of the Wavelog and Cloudlog.

>A migration from Cloudlog to Wavelog is easily possible if your existing Cloudlog installation is version 2.6.7 or earlier.

BTW my Cloudlog installation is 2.6.12 :)

**Reply by HB9HIL** [reply]
I'm sorry but they are incompatible due the changes.

You can still migrate from CL to WL even if you already on 2.6.12. Read the full article :)

**Reply by R1BLH** [reply]
Understandable.
Before real migration for me is interesting to test Wavelog (one bug I found already).
So I'm interested in database synchronization between Cloudlog and Wavelog. Doing this manually is too boring. :)
It is not a problem to do double logging for digi modes (like ft8 etc), but for modes that are manually logged this is a problem.

**Reply by HB9HIL** [reply]
To test Wavelog you also can try our demo: 

https://demo.wavelog.org

**Reply by R1BLH** [reply]
> To test Wavelog you also can try our demo:
> https://demo.wavelog.org

I already have it installed at home -- https://wavelog.cats-home.net

**Reply by HB9HIL** [reply]
Hope you will be happy with it :)
